### ISSUE TITLE
could not find `Stage` in `usd`

### ISSUE THREAD

**Issue by kubo-von** [luke-titley/usd-rs#1]
Hello,
I realize this is early in development, but since it's mention that this stuff should work I was curious what might I be doing wrong.
When trying to run the example I get this error:
```
error[E0433]: failed to resolve: could not find `Stage` in `usd`
 --> src/main.rs:4:22
  |
4 |     let stage = usd::Stage::create_in_memory();
  |                      ^^^^^ could not find `Stage` in `usd`
```

in cargo.tml:
```
[dependencies]
usd = "0.0.2"
```
main.rs:
```
extern crate usd;

fn main() {
    let stage = usd::Stage::create_in_memory();
    stage.export();
    println!("Hello, world!");

}
```

PS: Thanks a lot for working on this crate! I've been stuck calling USD python API from Rust until now, which is slow and clumsy

**Comment by luke-titley** [comment]
hey, sorry for the slow reply.

if you run:
cargo doc --open

You'll at least get a view of the crate so you can see how it's structured.
It's very early days at the moment, but yes, you should be able to create a stage.

The namespace is usd::pxr::usd::Stage. Think of it like the python modules where everything is under the pxr root module.

You don't need to specify 'extern crate usd' anymore in rust. So you can remove that line.

Luke

Sent with [ProtonMail](https://protonmail.com/?utm_campaign=ww-all-2a-mail-pmm_mail-protonmail_signature&utm_source=proton_users&utm_medium=cta&utm_content=sent_with_protonmail_secure_email) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by luke-titley** [comment]
Maybe the best example would be:
```
use usd::pxr::usd;
use std::ffi::CString;

fn main() {
    let filepath = CString::new("exported_here.usda").unwrap();
    let stage = usd::Stage::create_in_memory(usd::stage::desc::CreateInMemory::default());
    stage.export(filepath.as_c_str());
}
```

**Comment by kubo-von** [comment]
Thanks Luke!
I realized I could have used _crago doc_ afterwards to have a look inside, but still couldn't get it to work with 0.2.0 release.
Finally I tried to download and build from github, there I had to change the mods in:
`usd-rs/usd-rs/src/pxr/usd/mod.rs`
to be public, before building, and after that it works! the stage is created and saved to file.
Thanks a lot! I can start experimenting now. :)
